app/views/foods/new.html.erb

```diff
@@ -0,0 +1,46 @@
+<h2 class="mt-4 text-center">Add New Food</h2>
+<div class="row mt-5">
+  <div class="col-md-6 offset-md-3">
+    <div class="card">
+      <div class="card-body">
+        <%= form_with model: @food, url: foods_path, class: 'flex flex-col items-center w-6/12 mx-auto' do |form| %>
+          <% if @food.errors.any? %>
+            <div id="error_explanation" class="bg-red-50 text-red-500 px-3 py-2 font-medium rounded-lg mt-3">
+              <h2><%= pluralize(food.errors.count, "error") %> prohibited this food from being saved:</h2>
```

Comment:
- [ ] [OPTIONAL]
The app is throwing an error whiles creating 2 or more food items. Kindly improve upon this aspect.

**Adding multiple food error:**
![image](https://user-images.githubusercontent.com/12745474/219803332-a718f823-692c-46ed-bc20-107582b26d17.png)

Reply:
Resolved